multiscanner/distributed/api.py

```diff
@@ -1065,7 +1066,16 @@ def _main():
         )
         ms_process.start()
 
-    app.run(host=api_config['api']['host'], port=api_config['api']['port'])
+    in_docker = os.getenv("PRODUCTION", False)
+    
+    if not in_docker:
```

Comment:
Note that if the environment variable `"PRODUCTION"` is not set. The return value from line 1069 `os.getenv("PRODUCTION", False)` is `False`. The current check here (L1071) will evaluate to `True` and be executed which I think is **not** the intended behavior.

Reply:
The other part to this comment and also seen in other places is that its only checking for the presence of the environment variable, but not checking its value. Thus `ENV PRODUCTION No` would still execute the portion of the code meant for production.

Reply:
Thank you for the comments- you are correct. Will fix the code to work as intended.

Reply:
Closing to pull from working branch.